test/logic_value_test.dart

```diff
@@ -493,29 +493,30 @@ void main() {
           () => LogicValue.ofString('0000') - LogicValue.ofString('000100'),
           throwsA(isA<Exception>()));
     });
-    test('muldiv', () {
```

Comment:
Add modulo in addition, instead of replacing multiplication and division tests.

Reply:
Moved into arithmetic operations group